src/orchestration/Orchestration.ts

```diff
@@ -359,26 +323,29 @@ export class Orchestration {
                 id: applicationId,
                 version: applicationVersion
             },
-            this.config
+            this.getFullConfig()
```

Comment:
what needs to be fixed in a followup :-(

If I try making a shallow copy of the orchestration config, everything breaks (which really should not happen)